packages/nextra-theme-docs/src/components/sidebar.tsx

```diff
@@ -406,8 +406,8 @@ export function Sidebar({
           showSidebar ? 'md:_w-64' : 'md:_w-20',
           asPopover ? 'md:_hidden' : 'md:_sticky md:_self-start',
           menu
-            ? 'max-md:[transform:translate3d(0,0,0)]'
-            : 'max-md:[transform:translate3d(0,-100%,0)]'
+            ? 'max-md:[transform:translate3d(0,0,0)] max-md:_duration-200'
+            : 'max-md:[transform:translate3d(0,-100%,0)] max-md:_duration-200'
```

Comment:
I feel this toooo fast I prefer to remove this `translate3d` in v4, can we keep the current behaviour for v3?

Reply:
Fair enough!

Reply:
why closing? 🙂

Reply:
I might have misunderstood.
So we want to keep things as they are for current v3, but revisit for v4?

Reply:
I am fine with reducing the duration of the hamburger menu and removing translate3d in v4. Does that sound good to you?

Reply:
changed to 300ms, which seems to be a good middle.
Feel free to tweak the numbers if you feel it's still too fast.